bin/aeproject/utils.js

```diff
@@ -31,7 +46,40 @@ const copyFileOrDir = (sourceFileOrDir, destinationFileOrDir, copyOptions = {})
   fs.copySync(sourceFileOrDir, destinationFileOrDir, copyOptions)
 }
 
+const getFiles = async function (directory, regex) {
```

Comment:
Fix **test.js** file to use the function from the **utils** file instead the implementation in it

Reply:
Done